Teams.Tests/ManageTasksServiceTest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using MockQueryable.Moq;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Teams.Data;
+using Teams.Models;
+using Teams.Security;
+using Teams.Services;
+using Task = Teams.Models.Task;
+
+namespace Teams.Tests
+{
+    [TestFixture]
+    class ManageTasksServiceTest
+    {
+        private Mock<IRepository<Task, int>> _tasksRepository;
+        private ManageTasksService _manageTasksService;
+
+        [SetUp]
+        public void Setup()
+        {
+            _tasksRepository = new Mock<IRepository<Task, int>>();
+            _manageTasksService = new ManageTasksService(_tasksRepository.Object);
+            var mock = getFakeDbTasks().AsQueryable().BuildMock();
+            _tasksRepository.Setup(x => x.GetAll()).Returns(mock.Object);
+        }
+
+        [Test]
+        public async System.Threading.Tasks.Task GetAllTasksInTeamAsyncReturnListTasks_ReturnListTasks()
+        {
+            //Arrange
+            int team_id = 1;
```

Comment:
also, I missed it before, but better to use not snake case, please read doc here: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

should be teamId, here a lot of places to change.

snake case fine to use with consts, enum values. names for tests

Reply:
please take a look into it